src/config.rs

```diff
@@ -502,6 +700,19 @@ impl HookBuilder {
 
     /// Install the given Hook as the global error report hook
     pub fn install(self) -> Result<(), crate::eyre::Report> {
+
+        // XXX I'm not 100% sure if the following is the ideal way to handle these failures (however, I assume it is).
```

Comment:
As far as I can remember this is how we handle install errors in `eyre` and looks right to me
```suggestion
```